service/src/main/java/com/derpgroup/derpwizard/resource/AlexaResource.java

```diff
@@ -94,10 +95,11 @@ public SpeechletResponseEnvelope doAlexaRequest(@NotNull @Valid SpeechletRequest
       card.setTitle("Alexa + DERP Wizard");
 
       @SuppressWarnings("unchecked")
-      VoiceOutput<SpeechletResponse> voiceOutput = (VoiceOutput<SpeechletResponse>) VoiceMessageFactory.buildOutputMessage(builder.build(), InterfaceType.ALEXA);
-      SpeechletResponse speechletResponse = (SpeechletResponse) voiceOutput.getImplInstance();
+      VoiceOutput<SsmlOutputSpeech> voiceOutput = (VoiceOutput<SsmlOutputSpeech>) VoiceMessageFactory.buildOutputMessage(builder.build(), InterfaceType.ALEXA);
```

Comment:
We should add some unit tests for AlexaResource to catch these types of mistakes.